src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -191,5 +218,61 @@ private void CaptureAsyncExceptions(Task task)
                 }
             });
         }
+
+        private static class Log
```

Comment:
Nit: any chance this could go in a separate file?

Reply:
It's apparently an intentional pattern. To quote, @rynowak 

> Everything is solved with more nested classes.

Reply:
Nested classes and regions. Both more useful than inheritance